### DISCUSSION TITLE
A GitHub.io Hosted Ethereum Exploit is robbing people $$$$, How can we get it removed?

### DISCUSSION THREAD

**Discussion by SymonDT** [community/community#27530] [New to GitHub] [answered]
I want to get the attention of GitHub support here because there is a scammer who is exploiting our great platform in robbing innocent people. Different cases have been reported that the scammer is using YouTube to spread a maliciously coded scam hosted on this GitHub.io platform. 
In his video, he instructs his viewers on how to use **GitHub.io hosted** **[Remix IDE v0.21.4](https://remixethereumcompiler.github.io/#optimize=true&runs=200&evmVersion=null&version=soljson-v0.6.6+commit.6c089d02.js)** running a Solidity Compiler [File: Bot.Sol] by adding his own custom code to create a new contract.

One of his victims spoke about this scam [here](https://axeetech.com/github-io-hosted-ethereum-exploit/).

So I request to this community to make it sure to get rid of such evils from our platform.

**Comment by ettaboyle** [comment, marked as answer]
Hi @SymonDT 👋🏻     
We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

[Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
* [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
* [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
* [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

 You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
